### PR TITLE
US 8: Implement POST /api/users

### PR DESCRIPTION
**Pull request by RogelioLopez4th**
controllador,servicio,repositorio

### CODE REVIEW

File: Internship.API/Controllers/UsersController.cs [hunk 1]
```diff
@@ -11,5 +11,18 @@ namespace Internship.API.Controllers
...
+        private readonly UserService _userService;
+
+        public UsersController(UserService userService)
+        {
+            _userService = userService;
+        }
+                [HttpPost]
+        public ActionResult<User> Create(User user)
+        {
+            _bookService.Create(user);
```
Comment (fasaga) [inline Internship.API/Controllers/UsersController.cs]: This variable _bookService does not exist, change it, use the correct one

File: Internship.API/Services/UsersService.cs [hunk 2]
```diff
@@ -8,5 +8,36 @@ namespace Internship.API.Services
 {
     public class UsersService : IUsersService
     {
+        private readonly IMongoCollection<User> _user;
+
+        public BookService(IBookstoreDatabaseSettings settings)
```
Comment (fasaga) [inline Internship.API/Services/UsersService.cs]: This constructor name is incorrect change it, IBookstoreDatabaseSettings is also incorrect

File: Internship.API/Services/UsersService.cs [hunk 3]
```diff
@@ -8,5 +8,36 @@ namespace Internship.API.Services
...
+        {
+            var client = new MongoClient(settings.ConnectionString);
+            var database = client.GetDatabase(settings.DatabaseName);
+
+            _user = database.GetCollection<User>(settings.BooksCollectionName);
```
Comment (fasaga) [inline Internship.API/Services/UsersService.cs]: BooksCollectionName is incorrect should UsersCollectionName

File: Internship.API/Controllers/UsersController.cs (hunk 1 above)
Comment (RogelioLopez4th) [reply Internship.API/Controllers/UsersController.cs]: I'm sorry I changed it

File: Internship.API/appsettings.json [hunk 4]
```diff
@@ -1,8 +1,8 @@
 {
     "SourceSCDatabaseSettings": {
-        "UsersCollectionName": "Users",
+        "UsersCollectionName": "cli",
         "ConnectionString": "mongodb://localhost:27017",
-        "DatabaseName": "SourceSCDatabase"
+        "DatabaseName": "Users"
```
Comment (fasaga) [inline Internship.API/appsettings.json]: database name is "SourceSCDatabase"

File: Internship.API/appsettings.json [hunk 5]
```diff
@@ -1,8 +1,8 @@
 {
     "SourceSCDatabaseSettings": {
-        "UsersCollectionName": "Users",
+        "UsersCollectionName": "cli",
```
Comment (fasaga) [inline Internship.API/appsettings.json]: user collection name is "Users"

**Pull request merged**